presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClientModule.java

```diff
@@ -66,7 +66,7 @@ protected void setup(Binder binder)
         binder.bind(PhoenixClient.class).in(Scopes.SINGLETON);
         binder.bind(JdbcClient.class)
                 // TODO support JMX stats collection for phoenix connector
-                .annotatedWith(InternalBaseJdbc.class)
+                .annotatedWith(StatsCollecting.class)
```

Comment:
why stats-collecting? the bound object does not collect any stats.

here we want something like `@TheClient`.
Maybe _the client_ should be simply bound to `.bind(JdbcClient.class)`, without any annotation being part of the key?